env/admin.env

```diff
@@ -0,0 +1,66 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# APP
+FUNCTION_NAME=admin-service
+MAX_CPU=1
+NODE_ENV=dev
+
+# FASTIFY
+HOST=0.0.0.0
+PORT=3100
+
+# APM
+APM_SERVICE_NAME=tms-service
+APM_URL=http://apm:8200
+APM_SECRET_TOKEN=
+APM_ACTIVE=false
+
+# DB
+# # REDIS
+REDIS_DB=0
+REDIS_SERVERS=[{"host":"redis", "port":6381}]
```

Comment:
And this, though this whole file should map to the .template.env. This is still a copy of the tms service .env.